jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/TargetDirectory.java

```diff
@@ -130,19 +135,22 @@ static String createMyDirectory(String myVmId, boolean preserveId) throws IOExce
 	 */
 	static boolean deleteMyFiles() {
 		if (LOGGING_DISABLED != loggingStatus) {
-			IPC.logMessage("deleting my files"); //$NON-NLS-1$
+			IPC.logMessage("deleting my files: attachInfo files and attachNotificationSync files."); //$NON-NLS-1$
 		}
 
 		if ((null != advertisementFileObject) && advertisementFileObject.delete()) {
+			IPC.logMessage("deleted ", advertisementFileObject.getAbsolutePath());
 			advertisementFileObject = null;		
 		} else {
 			return false;
 		}
 		if ((null != syncFileObject) && syncFileObject.delete()) {
+			IPC.logMessage("deleted ", syncFileObject.getAbsolutePath());
 			syncFileObject = null;
 		} else {
 			return false;
 		}
+		IPC.logMessage("deleted my files with success.");
```

Comment:
```suggestion
		if (LOGGING_DISABLED != loggingStatus) {
			IPC.logMessage("deleted my files with success."); //$NON-NLS-1$
		}
```

Reply:
If I'm correct, all `IPC.logMessage` methods check logging level before print log message.
So, it's not necessory to check log level.

Reply:
https://github.com/eclipse-openj9/openj9/blob/cd8db74f1d43b37e3bdbd1c72f9e120d18047634/jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/IPC.java#L269-L288
`isLoggingEnabled()` might be `synchronized` during bootstrapping, adding an explicit check avoids such synchronization at fast code path.